### PR TITLE
Dependency update and accessibility improvement

### PR DESCRIPTION
**Pull request by jalezi**
All dependencies updated except `react-leaflet`.

Remove `react-i18next`. A little bit smaller build.

Not sure why but I had some problems with active links.

### CODE REVIEW

**Comment by jalezi** [comment]
@miaerbus @bananica @stefanb @lukarenko 

Če se komu da uč vržt, če vse dela 😺

**Comment by miaerbus** [comment]
Bravo, meni zgleda 👌

**Review by stefanb — APPROVED** [review]
:shipit:

File: package.json [hunk 1]
```diff
@@ -65,7 +65,6 @@
     "webpack-bundle-analyzer": "^4.7.0"
   },
   "engines": {
-    "node": ">= 17",
```
Comment (stefanb) [inline package.json]: To morda ni več potrebno po merganju:
- https://github.com/sledilnik/zdravniki/pull/293

**Pull request merged**